app/assets/stylesheets/govuk_publishing_components/components/_step-by-step-nav.scss

```diff
@@ -125,6 +126,10 @@ $top-border: solid 1px govuk-colour("mid-grey", $legacy: "grey-3");
 
   &:focus {
     @include step-nav-chevron-focus-state;
+
+    .gem-c-step-nav__button-text {
+      text-decoration: none;
+    }
```

Comment:
👋🏼   

You've mentioned 

> This change also includes a minor tweak to focus with hover so that the thick underline doesn't interfere visually with the small focus state.

I'm in two minds about this inconsistency for a button + focus + hover. However I understand what you mean about it being crowded.  I'd be tempted to increase the padding to enlarge the focus area to factor in the larger underline.

![May-26-2021 13-02-31](https://user-images.githubusercontent.com/71266765/119656488-a9b04880-be22-11eb-9727-ab9167db371d.gif)
vs
https://components.publishing.service.gov.uk/component-guide/step_by_step_nav

IE11 for ref
![May-26-2021 12-55-14](https://user-images.githubusercontent.com/71266765/119657338-ae293100-be23-11eb-9fb3-2d64050b48c5.gif)

----

Not related but i've noticed with the numbers on the side a hover is present but only while no "actively" selected and / black

Reply:
What do you think of this?

| Default | Hover + focus |
| --- | --- |
| ![Screenshot 2021-05-26 at 16 30 11](https://user-images.githubusercontent.com/64783893/119688297-c6a74480-be3f-11eb-84a5-287213d606fa.png) | ![Screenshot 2021-05-26 at 16 29 57](https://user-images.githubusercontent.com/64783893/119688336-ceff7f80-be3f-11eb-8f22-0e2e36fd7f55.png) |

I was concerned that it'd be too spacious but it's not too bad and is a little less crowded. The use case of focused _and_ underlined is also very slim IMO. If you're happy with this I can also apply this to the accordion and update the PR description.

Reply:
>The use case of focused and underlined is also very slim IMO.

Agreed, I'm also leaning towards the consistency aspect of the other buttons + focus + hover as well. 

> If you're happy with this I can also apply this to the accordion and update the PR description. 

It's probably also worth noting, reviewing the Design System, I think the inverse is probably more true with elements _not_ getting any hover change once focused.  (I know the general focus state in this context is being looked at in parallel)

So for the context of this link update keeping as is would be all good rn - thanks @owenatgov.

Reply:
Personally I don't like the focus also having the underline. But I'm one person. Can we consult a designer on this?